docs/hour-of-code-2022.html

```diff
@@ -136,20 +156,26 @@ <h1>
             </section>
             <section class="featured-list">
                 <div class="container container-grid">
-                    <div class="card">
+                    <div 
+                    class="card" 
+                    onClick="launchPage('http://www.google.com', 'hourofcode2022.skillmap.open.whackEmAll')">
```

Comment:
Are we launching google.com here? Why?

Reply:
If these are placeholder URL, @kiki-lee should have the right URL. Otherwise, point to bing.com?

Reply:
And a note: Whack-em-All has been changed to Whack-the-Mole.

- https://aka.ms/whack-the-mole
- https://aka.ms/balloon
- https://aka.ms/targets

and the teacher lesson plan is at 

- https://aka.ms/HoC-22-Lesson

Reply:
@abchatra these changes have been made